lib/galaxy/config/__init__.py

```diff
@@ -291,16 +288,16 @@ def _process_config(self, kwargs):
         self.user_tool_section_filters = listify(kwargs.get("user_tool_section_filters", []), do_strip=True)
         self.has_user_tool_filters = bool(self.user_tool_filters or self.user_tool_label_filters or self.user_tool_section_filters)
 
-        self.tour_config_dir = os.path.join(self.root, kwargs.get("tour_config_dir", "config/plugins/tours"))
-        self.webhooks_dirs = os.path.join(self.root, kwargs.get("webhooks_dir", "config/plugins/webhooks"))
+        self.tour_config_dir = os.path.join(self.root, self.tour_config_dir)
+        self.webhooks_dirs = os.path.join(self.root, self.webhooks_dir)
```

Comment:
I just noticed that:
- `webhooks_dir` can be a comma-separated list of directories, so resolving a list w.r.t `self.root` would affect only the first element of the list.
- the assigned attribute `self.webhooks_dirs` has a final `s`, while the config property doesn't

Given that each element of the list is resolved w.r.t. `self.root` any way in https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/util/__init__.py#L1584 , I'd suggest to remove this line:

```suggestion
```

and apply this one-line patch:
```patch
diff --git a/lib/galaxy/app.py b/lib/galaxy/app.py
index bd1de55caa..726c3380f3 100644
--- a/lib/galaxy/app.py
+++ b/lib/galaxy/app.py
@@ -155,7 +155,7 @@ class UniverseApplication(config.ConfiguresGalaxyMixin):
         # Tours registry
         self.tour_registry = ToursRegistry(self.config.tour_config_dir)
         # Webhooks registry
-        self.webhooks_registry = WebhooksRegistry(self.config.webhooks_dirs)
+        self.webhooks_registry = WebhooksRegistry(self.config.webhooks_dir)
         # Load security policy.
         self.security_agent = self.model.security_agent
         self.host_security_agent = galaxy.model.security.HostAgent(
```

Reply:
Would you mind if I do this as a separate PR? I can do it right now - will take a couple of minutes.. 
The reason I'm asking is (1) it's not fixing something this PR modified (so the potential list of such edits is very long); and (2) it's ~20 minutes away from passing the last test after which it can be merged - then I can finally rebase 8689. If I make this change, it'll launch all the tests all over again - and that'll take hours.

Reply:
Sure, you can address all 3 remaining comments in #8689 or a separate PR.

Reply:
Thank you! Will do.

Reply:
I believe the same applies to `tour_registry` (except the trailing 's'). Same thing: comma-separated list, resolved in the same way. I'm changing both.